test/WireMock.Net.Tests/WireMockServerTests.cs

```diff
@@ -133,6 +136,47 @@ public async Task WireMockServer_WithCorsPolicyOptions_Should_Work_Correct()
 }
 #endif
 
+#if NET461 || NETSTANDARD1_3 || NETCOREAPP3_1 || NET5_0 || NET6_0 || NET7_0
+    [Fact]
+    public async Task WireMockServer_WithRequiredClientCertificates_Should_Work_Correct()
+    {
+        // Arrange
+        var settings = new WireMockServerSettings
+        {
+            ClientCertificateMode = ClientCertificateMode.RequireCertificate,
+            SkipClientCertificateValidation = true,
+            UseSSL = true,
+        };
+
+        using var server = WireMockServer.Start(settings);
+
+        server.Given(Request.Create().WithPath("/*"))
+            .RespondWith(Response.Create().WithCallback(message => new ResponseMessage
+            {
+                StatusCode = message.ClientCertificate?.Thumbprint == "2E32E3528C87046A95B8B0BA172A1597C3AF3A9D"
+                    ? 200
+                    : 403
+            }));
+
+        var certificates = new X509Certificate2Collection();
+        certificates.Import("client_cert.pfx", "1234", X509KeyStorageFlags.Exportable);
```

Comment:
Can you please add a readme onhow to generate that certificate?

Reply:
I followed this https://learn.microsoft.com/en-us/aspnet/core/security/authentication/certauth?view=aspnetcore-7.0#create-certificates-in-powershell

Where should I add the readme; at the root of the test project, or in the one at the root of the repo?

Should I add the instructions directly, or a link to Microsoft's doc is sufficent?

Reply:
Create a readme in the same folder where that test client certificate is located.

Linking to website is ok, but if the number of instructions are limited, just add these to the readme.